gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/ApiServiceImpl.java

```diff
@@ -3092,4 +3093,55 @@ private void triggerNotification(String apiId, ApiHook hook, ApiEntity apiEntity
             notifierService.trigger(hook, apiId, new NotificationParamsBuilder().api(apiEntity).user(userService.findById(userId)).build());
         }
     }
+
+    @Override
+    public Map<String, Long> countPublishedByUserGroupedByCategories(String userId) {
+        ApiCriteria criteria = new ApiCriteria.Builder().visibility(PUBLIC).lifecycleStates(List.of(ApiLifecycleState.PUBLISHED)).build();
```

Comment:
I wonder if the PUBLIC visibility criteria is relevant here.
Are private APIs visible on portal, and should they appear in count ?

(anyway, seems conform to findApisByUser)

Reply:
As you noticed I went with what was used in findApisByUser

My understanding is

1. Fetch PUBLIC API list (as access is granted to any one)
2. Fetch the rest of the list according to my user & group memberships

So I guess the first request is legit but still, I'm assuming this from the code I found in `findApisByUser`

Reply:
Makes sense. Thanks. :+1: